manual/manual/allfiles.etex

```diff
@@ -80,15 +78,15 @@ and as a
 \input{stdlib-blurb.tex}
 \input{compilerlibs.tex}
 \input{libunix.tex}
-\input{libnum.tex}
 \input{libstr.tex}
 \input{libthreads.tex}
-\input{libgraph.tex}
+%\input{libgraph.tex}
 \input{libdynlink.tex}
-\input{libbigarray.tex}
+%\input{libbigarray.tex}
+\input{libnum.tex}
```

Comment:
if I understand correctly, `libnum.tex` has shifted purpose from being about Num to a "lost and found" section or something. I think it would be better to rename the file to clarify this change of semantics.

Reply:
Yes, this is to come... See my PR description.